src/server/eval/unary_operation_node.cpp

```diff
@@ -36,7 +36,7 @@ UnaryOperationNode::UnaryOperationNode(op::UnaryOperationEnum operation, const n
 string UnaryOperationNode::eval(GeoObject* obj) const
 {
 	// This is needed beacase of MSVC
-	assert(false);
```

Comment:
year but shouldn't do the function something,... like executing an operation of op::UnaryOperationEnum and evaluate the child node?